package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@duckduckgo/content-scope-scripts",
-  "version": "1.1.2",
```

Comment:
We do versioned releases on this repo - https://github.com/duckduckgo/content-scope-scripts/releases , is this `version` field not useful at all?

Reply:
It's useful if we automate it and keep it up to date.
But also I think it only becomes useful if we publish to npm; we can add back in as/when/if we do that.